x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/use_upgrade_with_conflicts_modal/translations.tsx

```diff
@@ -41,202 +43,139 @@ export const UPGRADE_RULES_WITH_CONFLICTS = (numOfRules: number) =>
-  </>
-);
-
-export const ONLY_RULES_WITH_NON_SOLVABLE_CONFLICTS = (numOfRules: number) => (
+export const RULES_WITH_NON_SOLVABLE_CONFLICTS_GUIDANCE = (
+  numOfRulesWithNonSolvableConflicts: number
+) => (
   <FormattedMessage
-    id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.onlyNonSolvableConflicts"
-    defaultMessage="{numOfRulesStrong} selected {numOfRules, plural, =1 {rule has} other {rules have}} unresolved conflicts. Rules with unresolved conflicts can’t be bulk-updated. You must manually fix their conflicts before updating them."
-    values={{ numOfRules, numOfRulesStrong: <strong>{numOfRules}</strong> }}
+    id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.rulesWithNonSolvableConflictsGuidance"
+    defaultMessage="{numOfRulesWithNonSolvableConflictsStrong} {numOfRulesWithNonSolvableConflicts, plural, =1 {rule} other {rules}} with unresolved {numOfRulesWithNonSolvableConflicts, plural, =1 {conflict} other {conflicts}} can’t be updated until you fix the {numOfRulesWithNonSolvableConflicts, plural, =1 {conflict} other {conflicts}}."
+    values={{
+      numOfRulesWithNonSolvableConflicts,
+      numOfRulesWithNonSolvableConflictsStrong: (
+        <strong>{numOfRulesWithNonSolvableConflicts}</strong>
+      ),
+    }}
   />
 );
 
-export const ONLY_RULES_WITH_CONFLICTS = ({
-  numOfRulesWithSolvableConflicts,
-  numOfRulesWithNonSolvableConflicts,
-}: {
-  numOfRulesWithSolvableConflicts: number;
-  numOfRulesWithNonSolvableConflicts: number;
-}) => (
-  <>
-    <FormattedMessage
-      id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.onlyRulesWithConflicts"
-      defaultMessage="Selected rules have conflicts. To safely update the rules, we recommend addressing the conflicts from the rule's update flyout."
-    />
-    <br />
-    <br />
-    <FormattedMessage
-      id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.unresolvedConflictsCanNotBeBulkUpdated"
-      defaultMessage="{numOfRulesWithNonSolvableConflictsStrong} out of the selected {numOfRulesStrong} {numOfRules, plural, =1 {rule} other {rules}} with unresolved conflicts can’t be bulk-updated. You must manually fix their conflicts before updating them."
-      values={{
-        numOfRules: numOfRulesWithSolvableConflicts + numOfRulesWithNonSolvableConflicts,
-        numOfRulesStrong: (
-          <strong>{numOfRulesWithSolvableConflicts + numOfRulesWithNonSolvableConflicts}</strong>
-        ),
-        numOfRulesWithNonSolvableConflictsStrong: (
-          <strong>{numOfRulesWithNonSolvableConflicts}</strong>
-        ),
-      }}
-    />
-    <br />
-    <br />
-    {PROCEED_WITH_NO_CONCERNS(numOfRulesWithSolvableConflicts)}
-  </>
+const ACCEPT_SOLVABLE_CONFLICTS_WARNING = i18n.translate(
+  'xpack.securitySolution.detectionEngine.upgradeConflictsModal.acceptSolvableConflictsWarning',
+  {
+    defaultMessage:
+      'Only choose this option if you’re comfortable accepting the fixes Elastic suggested.',
```

Comment:
@nastasha-solomon this reads a bit weird to me but I can't really figure out why. Any thoughts? I don't feel strongly enough about it to need a change, just wondering

Reply:
@dplumlee have you had a chance to ruminate on this a bit more? Do you think this suggestion would be unclear or confusing for users?